exporters/ostream/src/metric_exporter.cc

```diff
@@ -117,14 +117,14 @@ void OStreamMetricExporter::printInstrumentationInfoMetricData(
   // sout_ is shared
   const std::lock_guard<opentelemetry::common::SpinLockMutex> locked(lock_);
   sout_ << "{";
-  sout_ << "\n  name\t\t: " << info_metric.scope_->GetName()
+  sout_ << "\n  scope name\t: " << info_metric.scope_->GetName()
```

Comment:
I was unsure if this should really be `scope` or `meter` (since the `meter` is what implicitly creates this scope). WDYT?

Reply:
I think `scope` is the right terminology in compliance with the specs.

Reply:
I think `scope` is the right terminology in compliance with the specs.